### ISSUE TITLE
add image alts for event featured images

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/cdh-web#168]
event pages currently don't have an `alt=` attribute on their featured image:
```html
<img property="schema:image" src="/media/uploads/events/poster_final_.jpg">
```

**Comment by thatbudakguy** [comment]
no alt text available for use, so just using a blank alt to skip the image entirely.

**Comment by rlskoeser** [comment]
Looks good. I think projects should have this fix also.
